spyglass/data_extractor/formation.py

```diff
@@ -313,13 +313,19 @@ def get_hosts(self, region, rack=None):
         control_hosts = device_api.zones_zone_id_control_nodes_get(zone_id)
         compute_hosts = device_api.zones_zone_id_devices_get(
             zone_id, type='KVM')
-
         hosts_list = []
+        genesis_set = False
         for host in control_hosts:
             self.device_name_id_mapping[host.aic_standard_name] = host.id
+            # The first control node is designated as genesis node
+            if genesis_set is False:
+                node_type = 'genesis'
+                genesis_set = True
+            else:
+                node_type = 'genesis'
```

Comment:
node_type should be genesis in else loop?

Reply:
This has been fixed. It should be controller